src/helpers.js

```diff
@@ -0,0 +1,121 @@
+/**
+ *Created by Mikael Lindahl on 2018-01-05
+ */
+
+"use strict";
+
+const TYPES = {
+    number: 'INTEGER',
+    string: 'TEXT',
+    boolean: 'BOOLEAN',
+    json: 'JSON'
+}
+
+/**
+ * Crete model definitions to be used when defining tables
+ * with the dapteres
+ *
+ * @param models
+ * @param datastores
+ * @returns {{}}
+ */
+function createDefinition(models, datastores) {
+
+    let definition = {};
+
+    for (let key1 in models) {
+
+        let model = models[key1];
+        let adapter = datastores[model.datastore].adapter;
+
+        if (!definition[adapter]){
+
+            definition[adapter]={}
+
+        }
+
+        definition[adapter][key1]= {
+                config: JSON.parse(JSON.stringify(model.attributes)),
+                datastore: model.datastore
+            }
+
+
+        for (let key2 in model.attributes) {
```

Comment:
I would be import lodash and for consistency iterate by `forEach`. Code should be simplified for reading after

Reply:
Haha, my bad. Not to used with lodash. WIll have a look